search/geocoder.cpp

```diff
@@ -736,21 +734,21 @@ void Geocoder::ForEachCountry(vector<shared_ptr<MwmInfo>> const & infos, TFn &&
   }
 }
 
-void Geocoder::MatchRegions(BaseContext & ctx, RegionType type)
+void Geocoder::MatchRegions(BaseContext & ctx, Region::Type type)
 {
   switch (type)
   {
-  case REGION_TYPE_STATE:
+  case Region::TYPE_STATE:
     // Tries to skip state matching and go to cities matching.
     // Then, performs states matching.
     MatchCities(ctx);
     break;
-  case REGION_TYPE_COUNTRY:
+  case Region::TYPE_COUNTRY:
     // Tries to skip country matching and go to states matching.
     // Then, performs countries matching.
-    MatchRegions(ctx, REGION_TYPE_STATE);
+    MatchRegions(ctx, Region::TYPE_STATE);
     break;
-  case REGION_TYPE_COUNT: ASSERT(false, ("Invalid region type.")); return;
+  case Region::TYPE_COUNT: ASSERT(false, ("Invalid region type.")); return;
```

Comment:
Ok. It's up to you. But as you know LOG(LERROR, ...) has an important feature. Its records are seen even in release version. I guess sometimes it could be helpful.